ee/query-service/usage/manager.go

```diff
@@ -139,17 +139,11 @@ func (lm *Manager) UploadUsage() {
 
 	zap.S().Info("uploading usage data")
 
-	// Try to get the org name
 	orgName := ""
-	orgNames, err := lm.modelDao.GetOrgs(ctx)
-	if err != nil {
-		zap.S().Errorf("failed to get org data: %v", zap.Error(err))
-	} else {
-		if len(orgNames) != 1 {
-			zap.S().Errorf("expected one org but got %d orgs", len(orgNames))
-		} else {
-			orgName = orgNames[0].Name
-		}
+	// fetching orgname will result in error only when org is not set, so ignoring it.
+	orgNames, _ := lm.modelDao.GetOrgs(ctx)
```

Comment:
It must be returning something similar to `sql.NoRows` or something? I would check for that specific case to ignore instead of ignoring every kind of error.

Reply:
Thanks, have corrected it.

So basically if there are no rows it doesn't throw an error but since the err variable was initialized in the same block again with orgNames it caused the error. Have fixed and updated the description.